Library/ViewModels/RewardAddOnSelectionNoShippingViewModelTests.swift

```diff
@@ -853,9 +860,10 @@ final class RewardAddOnSelectionNoShippingViewModelTests: TestCase {
       self.loadAddOnRewardsIntoDataSourceAndReloadTableView.assertValues(
         [expected], "TableView does not reload again"
       )
-      self.configureContinueCTAViewWithDataQuantity.assertValues([0, 0, 0, 5, 5])
-      self.configureContinueCTAViewWithDataIsValid.assertValues([true, true, true, true, true])
-      self.configureContinueCTAViewWithDataIsLoading.assertValues([true, false, false, false, false])
+      self.configureContinueCTAViewWithDataQuantity.assertLastValue(5)
+      self.configureContinueCTAViewWithDataIsValid.assertLastValue(true)
+      self.configureContinueCTAViewWithDataIsLoading.assertLastValue(false)
+      XCTAssertEqual(self.configureContinueCTAViewWithDataPledgeAmount.lastValue??.string, "$60.00")
```

Comment:
Note: I've updated these and similar tests to only check the last value instead of asserting all values, since adding the total here about doubles the amount of value changes (any time bonus support or add-ons change, so does the total). I found just checking the last value to be more useful and more clear.